notesapi/v1/views.py

```diff
@@ -141,15 +141,30 @@ class AnnotationDetailView(APIView):
 
     UPDATE_FILTER_FIELDS = ('updated', 'created', 'user', 'consumer')
 
+    def _fetch(self, annotation_id, reply_id=None):
+        """
+        Annotation-type or Comment-type agnostic Note fetcher.
+
+        Returns a Note instance if found, None otherwise.
+        """
+        try:
+            if reply_id:
+                note = Note.comments.get(id=reply_id, parent_id=annotation_id)
+            else:
+                note = Note.objects.get(id=annotation_id)
+        except Note.DoesNotExist:
+            return None
+
+        return note
+
     def get(self, *args, **kwargs):  # pylint: disable=unused-argument
         """
         Get an existing annotation.
         """
-        note_id = self.kwargs.get('annotation_id')
+        print(self.kwargs)
```

Comment:
Thanks, I'll be sure this doesn't get into the final thing. Ignore the code for now. This is just a "Hey. There's an API. You can see it in a UI and stuff" sort of PR.